modular_citadel/code/modules/reagents/chemistry/recipes/fermi.dm

```diff
@@ -591,19 +591,21 @@
 /datum/chemical_reaction/fermi/zeolites
 	name = "Zeolites"
 	id = /datum/reagent/fermi/zeolites
-	results = list(/datum/reagent/fermi/zeolites = 5) //We make a lot!
+	results = list(/datum/reagent/fermi/zeolites = 5) //We make a lot! - But it's now somewhat dangerous, and needs a bit of uranium to catalyze the reaction
 	required_reagents = list(/datum/reagent/medicine/potass_iodide = 1, /datum/reagent/aluminium = 1, /datum/reagent/silicon = 1, /datum/reagent/oxygen = 1)
+	required_catalysts = list(/datum/reagent/uranium = 5)
```

Comment:
I suggest using gold rather then uranium, gold has some anti-rad  stuff going on. If were being "realistic" the uranium would not be catalysts at all as it I think would be destoryed by the zeolite and the zeolite would also be harmed as its getting bonded into clumps around uranium ions or something
https://www.sharrettsplating.com/blog/metal-plating-satellites-benefits-light-reflection/#:~:text=Reflectivity%3A%20Gold%20is%20a%20highly,0.35%20%C2%B5m%20than%20other%20metals.
^ Source for the gold reasoning

Also also gold in chems us only really used to make wealth items or a chem or two. Having kinda small uses in crafting and such, giving it more uses would be good. Uranium as a lot of uses already for level 4 stockparts, genes and loads of other things making it have a fair share already of uses well gold does not 
^ Game reasoning for using gold over uranium

Reply:
Mh, uranium doesn't exactly destabilize zeolites unless in high concentrations.
Given that it's something to purge (absorb / adsorb) rads, and no protect against exposure (reflect) rads, I do not think gold would be better realistically speaking.
The reaction isn't exactly too realistic anyways, but all of this is besides the point anyways.

Now, the thing of impact here, gameplay effect:
While yeah, not as many things use gold (stock parts, buncha printables) as opposed to uranium (stock parts, buncha printables, some chem stuff, viro), considering the low quantities required, aswell as it not being used up during the reaction (catalyst), it doesn't exactly matter how relevant the use is, only how hard it is to acquire 1 sheet, which gold and uranium are around the same difficulty, usually. Actually, for gold it's easier to acquire small reagent quantities, given there's some gold in the vault roundstart, but generally no uranium, but thats fairly minor nonetheless.
So, gonna keep this as uranium.